fdbserver/MoveKeys.actor.h

```diff
@@ -106,5 +106,7 @@ ACTOR Future<Void> removeKeysFromFailedServer(Database cx,
 // Directly removes serverID from serverKeys and keyServers system keyspace.
 // Performed when a storage server is marked as permanently failed.
 
+ACTOR Future<UID> maybeUpdateTeamMaps(Database cx, vector<UID> team);
```

Comment:
Please add comments describe this function.

```suggestion
ACTOR Future<UID> maybeUpdateTeamMaps(Database cx, std::vector<ptxn::StorageTeamID> team);
```

Reply:
These UID are for storage servers in the team, but I will replace team IDs with ptxn::StorageTeamID everywhere.